dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditAPIImpl.java

```diff
@@ -222,16 +224,12 @@ public PublishAuditStatus getPublishAuditStatus(String bundleId)
 		}
 	}
 
-	private final String SELECTSQLALL=
-			"SELECT * "+
-			"FROM publishing_queue_audit order by status_updated desc";
-
 	@Override
 	@CloseDBIfOpened
 	public List<PublishAuditStatus> getAllPublishAuditStatus() throws DotPublisherException {
 		try{
 			DotConnect dc = new DotConnect();
-			dc.setSQL(SELECTSQLALL);
+			dc.setSQL(SELECT_ALL_ORDER_BY_STATUSUPDATED_DESC);
```

Comment:
Please log Bundle ID data in CATCH block.

Reply:
we are not receiving any argument, not sure what to log